tests/test_db_containers.py

```diff
@@ -100,3 +101,13 @@ def connect():
         cursor = db.restaurants.find({"borough": "Manhattan"})
         for document in cursor:
             print(document)
+
+@pytest.mark.skip(reason="needs ms sql client libraries unavailable on Travis")
```

Comment:
How we can test it then?

Reply:
I'm not sure! I ran it locally and followed the example for Oracle here... I thought it wouldn't work in Travis because of [discussions like this one](https://travis-ci.community/t/add-service-for-mssql-server-databases/5423/6), but it might just work using sqlalchemy.